### DISCUSSION TITLE
Adding output_parameters to the Session

### DISCUSSION THREAD

**Discussion by XX-Yin** [AllenNeuralDynamics/aind-data-schema#1003] [Ideas]
**Is your feature request related to a problem? Please describe.**
Move my personal discussion with @saskiad to here. 

The flexible `output_parameters` only existed in the `StimulusEpoch`, which does not make sense. A lot of parameters are related to that session instead of a specific stimulus. We should also add the `output_parameters` to `session.py/Session`. 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by dbirman** [comment]
This is very old, but in any case this is now covered by the `Code` objects
